components/script/script_thread.rs

```diff
@@ -2143,8 +2083,6 @@ impl ScriptThread {
 
         // https://html.spec.whatwg.org/multipage/#the-end steps 3-4.
         document.process_deferred_scripts();
-
-        window.set_fragment_name(final_url.fragment().map(str::to_owned));
```

Comment:
Yes, as my experiment on firefox and chrome, both browsers do not scroll to the fragment in the url after resize; instead, they kind of remeber the previous scroll point and keep to that point after resize, so I guess it is appropriate to remove this mechanism, since this behavior is kind of weird; that said, I have not be able to find the specification of the behavior of resize.